source/io/EventLoopGroup.cpp

```diff
@@ -25,25 +24,25 @@ namespace Aws
             {
                 if (!m_lastError)
                 {
-                    aws_event_loop_group_clean_up(&m_eventLoopGroup);
+                    aws_event_loop_group_release(m_eventLoopGroup);
```

Comment:
Though `m_lastError` currently only refers to the `new` operation result, better to use
```
if (m_eventLoopGroup)
{
    release
}
```
or simply
```
release
```
if `aws_event_loop_group_release` has the protection.
in case later we add more operations and m_lastError doesn't necessarily identify the existence of underlying handle.